frontend-project/src/pages/cases/new/index.tsx

```diff
@@ -1,8 +1,19 @@
 import { PageHeaderWrapper } from '@ant-design/pro-layout';
 import { Button, Col, Card, Form, Input, Row, Select } from 'antd';
 import { connect } from 'dva';
-import React, { useEffect } from 'react';
+import React, { useEffect, FunctionComponent } from 'react';
 import { formatMessage, FormattedMessage } from 'umi-plugin-react/locale';
+import { User } from '@/models/users';
+
+interface Tag {
```

Comment:
to nie powinno przypadkiem trafić do `@/models/tags` (tak jak `User` jest w `@/models/users`)?